tests/test_grid.py

```diff
@@ -161,6 +160,10 @@ def test_add(self):
         np.testing.assert_array_equal(fd2.data_matrix[..., 0],
                                       [[3, 4, 5, 6], [4, 5, 6, 7]])
 
+        fd2 = fd1 + (lambda x: x)
```

Comment:
We would need to test in addition:
- Behaviour for functions of several variables.
- Behaviour for vector-valued functions.
- A combination of all the above.